Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -1536,4 +1533,41 @@ theorem multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multipl
     multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity hr ha]
 #align multiplicity_normalized_factors_equiv_span_normalized_factors_symm_eq_multiplicity multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multiplicity
 
+/-- The bijection between the set of prime factors of the ideal `⟨r⟩` and the set of prime factors
+  of `r` preserves `count` of the corresponding multisets. See
+  `multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity` for the version
+  stated in terms of multiplicity. -/
+theorem count_normalizedFactorsSpan_eq_count {r X : R} (hr : r ≠ 0) (hX : Prime X) :
```

Comment:
On second look, I don't think `normalizedFactorsSpan` makes sense as a name segment, so should be `normalizedFactors_span` in both cases.

Also, I think the name would make more sense if the conclusion was stated in the opposite direction.

Reply:
OK, I will change them to `normalizedFactors_span`, but concerning the "opposite direction", are you suggesting that the first (rsp. the second) theorem becomes `count_eq_normalizedFactorsSpan` and similarly with `'` ?

Reply:
In this direction I guess I'd go for `count_normalizedFactors_eq_count_normalizedFactors_span` which isn't so great - but I do think you want to include `span`. `count_normalizedFactors_span_eq_count` would work if you changed the conclusion. That might be a good idea anyway - the LHS right now seems simpler than the RHS.

Reply:
I agree with Ruben. Flipping the equality and naming the lemmas `count_span_normalizedFactors_eq`, `count_span_normalizedFactors_eq_of_normUnit` and `count_associates_factors_eq`.

Reply:
Thanks, done!